charts/tensorleap/Chart.lock

```diff
@@ -0,0 +1,18 @@
+dependencies:
+- name: ingress-nginx
+  repository: https://kubernetes.github.io/ingress-nginx
+  version: 4.1.2
+- name: elasticsearch
+  repository: https://helm.elastic.co
+  version: 7.10.2
+- name: kibana
+  repository: https://helm.elastic.co
+  version: 7.6.1
+- name: rabbitmq
+  repository: https://charts.bitnami.com/bitnami
+  version: 9.1.3
+- name: minio
+  repository: https://charts.min.io
+  version: 3.4.3
+digest: sha256:f2286d58c148d4c1a6c0175ad94eea6aace0100967f1a4847ad3dda46d67a381
```

Comment:
not sure what adding this file will cause (mostly worried about the digest, I don't know what goes in it)

Reply:
Fixed